screenshot-testing/src/main/java/com/stripe/android/screenshottesting/PaparazziRule.kt

```diff
@@ -101,6 +101,7 @@ class PaparazziRule(
             environment = detectEnvironment().run {
                 copy(compileSdkVersion = 33, platformDir = platformDir.replace("34", "33"))
             },
+            theme = "Theme.MaterialComponents",
```

Comment:
Curious why this is needed

Reply:
PrimaryButton isn't compose! And it requires this theme for xml! Otherwise the test crashes.